tests/erlang_tests/floorfloatovf.erl

```diff
@@ -23,13 +23,13 @@
 -export([start/0]).
 
 start() ->
-    to_int(id(id([1.0e+20, 0]))).
+    to_int(id(id([-1.0e+257, 0]))).
```

Comment:
```suggestion
    to_int(id(id([-1.0e+78, 0]))).
```
-10^78 is the smallest (in absolute value, or greatest) value that overflows.

Also, arguably, floor on a negative float and on a positive float may test different branches, especially with the negative and positive integers we now have.